### ISSUE TITLE
Improvement : Add an markable order for questions

### ISSUE THREAD

**Issue by stmolivier** [UJM-dev/ExoBundle#87]
It would be nice if the answer for a question (MCQ, ...) would have also an order
In a MCQ, you have  : 
1 - Answer 1  => TRUE
2 - Answer 2  => FALSE
3 - Answer 3  => TRUE
4 - Answer 4  => TRUE
But to have the full mark, you would have to give the correct order :  1 - 4 - 3

For instance : 
How to access you access a document on your desktop :
1 - Start your computer 
2 - Check your mail
3 - Double clic on your document
4 - Enter your password

**Comment by stefk** [comment]
Interesting, but shouldn't that be a separate question type ? Something like "Pick and sort" maybe... I think this might be less confusing for users, and would allow a distinct, better suited UI than plain MCQs.

**Comment by charlotte42** [comment]
Maybe with matching questions : 
![create1](https://cloud.githubusercontent.com/assets/11042666/10576140/62ff5660-7661-11e5-8085-690e0a2fafd6.png)
![questions](https://cloud.githubusercontent.com/assets/11042666/10576148/6e9dd442-7661-11e5-8636-21a337c81149.png)
![reponse](https://cloud.githubusercontent.com/assets/11042666/10576150/721e754a-7661-11e5-91bf-16ded7465539.png)

**Comment by stmolivier** [comment]
@stefk : I totally agree that it should be discussed beforehand. but still i'm not sure it should be another type of question altogether,This "order" could be seen as an option for existing question types (if not set, it's just a regular MCQ, matching question...)
@charlotte42 : yes, the matching question could be another type using it.
To a certain extent, the graphical could also be used (though, in that case, it would be much more complex, since the order would be the layer order)

This is just an idea, i don't know what the best way to do it would be.
